### PR TITLE
Clements -geom

### PR DESCRIPTION
**Pull request by sdkclements**
comfort: 4.5
comp: 5

### CODE REVIEW

File: geom.js [hunk 1]
```diff
@@ -3,6 +3,16 @@ class Rectangle {
     this.length = length;
     this.width = width;
   }
+  isSquare(){
+    this.length === this.width? true:false
```
Comment (amaseda) [inline geom.js:7]: Don't forget a `return` statement here. That way you could use this method in situations like this...

```js
if(squareInstance.isSquare()){
  // code that would run if it's a square
}

File: geom.js [hunk 2]
```diff
@@ -12,6 +22,23 @@ class Triangle {
     this.sideB = sideB;
     this.sideC = sideC;
   }
+  isEquilateral(){
+    (this.sideA === this.sideB && this.sideB === this.sideC)
```
Comment (amaseda) [inline geom.js:26]: Also would want a `return` statement here.

File: geom.js [hunk 3]
```diff
@@ -12,6 +22,23 @@ class Triangle {
...
+
+  }
+  isIsoceles(){
+    (this.sideA === this.sideB || this.sideA === this.sideC || this.sideB === this.sideC)
```
Comment (amaseda) [inline geom.js:30]: And a `return` here.

**Comment by sdkclements** [comment]
jeeze ruby made me forget my returns

**Comment by amaseda** [comment]
Ha figured that was the case.

**Pull request closed without merging**